charts/brigade/values.yaml

```diff
@@ -45,6 +45,11 @@ controller:
   # The default tag is the value of .Chart.AppVersion
   #tag:
   #pullPolicy: IfNotPresent
+  #workerResources:
+  #  limitsCPU:
```

Comment:
nit: do you think the structure of these values could be aligned with those used for the vcsSidecarResources in the brigade-project values.yaml?

(Meaning, `workerResources.limits.cpu`, `workerResources.limits.memory`, etc.)